RickLeinecker/Fortuna
# Added rendering of tank in armory

**Description**
Made tanks render in the armory. See #190 for details

Resolves #190

**Type of change**
Check options that are relevant.


**Steps to Verify Functionality**
Make a bulleted/numbered list of steps to verify this feature/bugfix. Include screenshots if necessary.
Looks like this:
![image](https://user-images.githubusercontent.com/18317476/77808834-6c57fb80-7063-11ea-9aa0-761b3009fb24.png)

**Final Checklist:**
Go down the list and check them off.
